moj_auth/tests/test_models.py

```diff
@@ -0,0 +1,51 @@
+from django.test import SimpleTestCase
+
+from .utils import generate_tokens
+from ..models import MojUser
+
+
+class UserPermissionsTestCase(SimpleTestCase):
+
+    def setUp(self, *args, **kwargs):
+        super(UserPermissionsTestCase, self).setUp(*args, **kwargs)
+
+        self.user = MojUser(5, generate_tokens(), {
+            'first_name': 'Sam',
+            'last_name': 'Hall',
+            'permissions': [
+                'can_has_cheezburger',
+                'right_to_party',
+                'access_mainframe'
```

Comment:
I guess, but I kinda feel like good code *should* be boring and predictable. If it's exciting, we're doing something wrong.

I only really thought about this because I had a moment of “wait, does this user have permission to open the pod bay doors?” when reading `test_has_perms_fails_if_any_absent`.